src/data.c

```diff
@@ -492,7 +492,7 @@ void fill_truth(char *path, char **labels, int k, float *truth)
             ++count;
         }
     }
-    if(count != 1) printf("Too many or too few labels: %d, %s\n", count, path);
+    // if(count != 1) printf("Too many or too few labels: %d, %s\n", count, path);
```

Comment:
this check is avoided to allow the ground truth to have more than one `1` in it.

Probably this is not the most elegant way to do this

Reply:
This line should be commented instead https://github.com/AlexeyAB/darknet/blame/master/src/data.c#L537 since https://github.com/AlexeyAB/darknet/commit/2a873f34485c75d44a346f92ba7dcf2e2aa57a15

Reply:
@kossolax can you clarify where we should comment? Line 537 points out a bracket only

Reply:
"Too many or too few labels" in the "fill_truth_smooth" function.

Reply:
I guess lines 539-548, which is all of `if (count != 1)` block.

Reply:
yes